gravity-forms/gw-save-and-continue-auto-load.php

```diff
@@ -166,7 +166,8 @@ public function is_editing_entry( $form_id ) {
 	public function is_applicable_form( $form ) {
 		$form_id = isset( $form['id'] ) ? $form['id'] : $form;
 
-		if ( $this->is_editing_entry( $form_id ) ) {
+		$form = GFAPI::get_form( $form_id );
+		if ( $this->is_editing_entry( $form_id ) || ! $form['save']['enabled'] ) {
 			return false;
 		}
```

Comment:
`rgars( $form, 'save/enabled' )` is likely the safer option here. It's what I see used in GF core.

I'd also group the variables like so:

```php
$form_id = isset( $form['id'] ) ? $form['id'] : $form;
$form    = GFAPI::get_form( $form_id );

if ( $this->is_editing_entry( $form_id ) || ! rgars( $form, 'save/enabled' ) ) {
	return false;
}
```